expression/builtin_string.go

```diff
@@ -1696,7 +1696,71 @@ type builtinExportSetSig struct {
 
 // See https://dev.mysql.com/doc/refman/5.6/en/string-functions.html#function_export-set
 func (b *builtinExportSetSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("export_set")
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	var (
+		bits         string
+		on           string
+		off          string
+		separator    = ","
+		numberOfBits = 64
+	)
+	arg0, err := args[0].ToInt64(b.ctx.GetSessionVars().StmtCtx)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	bits = strconv.FormatInt(arg0, 2)
+	on, err = args[1].ToString()
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	off, err = args[2].ToString()
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	switch len(args) {
+	case 4:
+		separator, err = args[3].ToString()
+		if err != nil {
+			return d, errors.Trace(err)
+		}
+	case 5:
+		separator, err = args[3].ToString()
+		if err != nil {
+			return d, errors.Trace(err)
+		}
+		arg4, err := args[4].ToInt64(b.ctx.GetSessionVars().StmtCtx)
+		if err != nil {
+			return d, errors.Trace(err)
+		}
+		if arg4 < 0 || arg4 > 64 {
+			arg4 = 64
```

Comment:
default value of arg4 is 64, so we can remove line 1739.
We can check if `arg4>=0 && arg4<64` here, 
if so, we set numberOfBits=int(args4).